Source/IntegrationTests/TestFiles/LitTests/LitTest/git-issues/git-issue-2064.dfy.expect

```diff
@@ -0,0 +1,35 @@
+git-issue-2064.dfy(36,9): Error: to be a refinement of abstract type 'UnitTestParent.U', type synonym 'OpaqueTypeMixedUp.U' must support the type characteristic "equality"
+git-issue-2064.dfy(37,9): Error: to be a refinement of abstract type 'UnitTestParent.V', type synonym 'OpaqueTypeMixedUp.V' must support the type characteristic "auto-initialization"
+git-issue-2064.dfy(51,9): Error: to be a refinement of abstract type 'UnitTestParent.T', type synonym 'Arrow.T' must support the type characteristic "no references"
+git-issue-2064.dfy(52,9): Error: to be a refinement of abstract type 'UnitTestParent.U', type synonym 'Arrow.U' must support the type characteristic "equality"
+git-issue-2064.dfy(53,9): Error: to be a refinement of abstract type 'UnitTestParent.V', type synonym 'Arrow.V' must support the type characteristic "auto-initialization"
+git-issue-2064.dfy(54,9): Error: to be a refinement of abstract type 'UnitTestParent.W', type synonym 'Arrow.W' must support the type characteristic "nonempty"
+git-issue-2064.dfy(60,9): Error: to be a refinement of abstract type 'UnitTestParent.U', type synonym 'ArrowGood.U' must support the type characteristic "equality"
+git-issue-2064.dfy(66,9): Error: to be a refinement of abstract type 'UnitTestParent.T', type synonym 'ArrowDirect.T' must support the type characteristic "no references"
+git-issue-2064.dfy(67,9): Error: to be a refinement of abstract type 'UnitTestParent.U', type synonym 'ArrowDirect.U' must support the type characteristic "equality"
+git-issue-2064.dfy(68,9): Error: to be a refinement of abstract type 'UnitTestParent.V', type synonym 'ArrowDirect.V' must support the type characteristic "auto-initialization"
+git-issue-2064.dfy(69,9): Error: to be a refinement of abstract type 'UnitTestParent.W', type synonym 'ArrowDirect.W' must support the type characteristic "nonempty"
+git-issue-2064.dfy(74,9): Error: to be a refinement of abstract type 'UnitTestParent.T', type synonym 'Class.T' must support the type characteristic "no references"
+git-issue-2064.dfy(76,9): Error: to be a refinement of abstract type 'UnitTestParent.V', type synonym 'Class.V' must support the type characteristic "auto-initialization"
+git-issue-2064.dfy(77,9): Error: to be a refinement of abstract type 'UnitTestParent.W', type synonym 'Class.W' must support the type characteristic "nonempty"
+git-issue-2064.dfy(81,10): Error: to be a refinement of abstract type 'UnitTestParent.T', class 'ClassDirect.T' must support the type characteristic "no references"
+git-issue-2064.dfy(83,10): Error: to be a refinement of abstract type 'UnitTestParent.V', class 'ClassDirect.V' must support the type characteristic "auto-initialization"
+git-issue-2064.dfy(84,10): Error: to be a refinement of abstract type 'UnitTestParent.W', class 'ClassDirect.W' must support the type characteristic "nonempty"
```

Comment:
```suggestion
git-issue-2064.dfy(84,10): Error: to be a refinement of abstract type 'UnitTestParent.W' declared `(00)`, class 'ClassDirect.W' must provably be non-empty, which is not possible for objects - you might want to refine `type V = ClassName?` so that `null` is a proof `V` is not empty
```